### ISSUE TITLE
FAN section not working properly?

### ISSUE THREAD

**Issue by phila82r** [jaschaknack/PolarisBiosEditor#37]
Hi there, awesome Polaris fork btw!

My only issue is with the FAN section. I tried many different settings for my `GB AORUS RX 580 4GB` card, and nothing changes the fan speed or threshold. I played with the Hysteresis, Sensitivity, and Acoustic Limit, putting them at max, misc or low settings - no changes.

I lowered all Temps (C) and PWM (%), nothing changes. I maxed them all, again, no changes. Just wondering if I'm doing something wrong?

I'm trying to get the fans working at ~40% to keep the GPU around 55C. For this card and through the AMD Radeon Settings, the max RPM is 3700, the max Acoustic Limit is 1175.

The rest is working great for me. Let me know if there is anything I can do to help. Cheers!

**Comment by charlesrg** [comment]
Did you set temp target ?

**Comment by phila82r** [comment]
Yep, I tried a target temp at 50 and 80. No changes. I'm using Claymore's miner and just figured out how to manage the min fan speed and gpu temp from command line, so that works in the meantime.

**Comment by charlesrg** [comment]
What do you mean at 50 and 80 ? Did you set the target temp to 50 ? If yes
the fan would kick in and stay at 100% most of the time trying to stay at
the 50 degrees range.

**Comment by MKOLYADA** [comment]
I also confirm this fan issue for RX 550. The rest works fine.

**Comment by phila82r** [comment]
I meant I did set the target temp at 50 to test it, and the gpu was above and the fans weren't even spinning much. Then I tried 80 as target temp, but had the same result. I updated my AMD drivers + Claymore's and it looks like it is working fine now for my RX 580.

**Comment by tfjield** [comment]
It looks to me like it's not writing the FAN section details to the file.  I modified the Fuzzy Fan Mode from 1 to 0, and saved the BIOS file.  I reloaded it and the value was back to 1.  I compared the MD5 checksum of the original and the "modded" BIOS files and they were identical.

**Comment by sambul13** [comment]
Can the dev add 0DB control and target temp to the package? AMD GPUs with 0DB enabled constantly start / stop the fans at idle to keep T around 45C. Many folks want the fans to spin slowly at GPU idle instead of ON/Off endless cycle accompanied by the board ongoing relay clicks. Pls expose Ts and controls for 0DB feature.

**Comment by georgenicholas** [comment]
I can confirm what tfjield said, 1.6.7 does not actually save the fuzzy fan value if changed.  I also tried it in 1.6.2 and it did save the fuzzy fan value.  However, when flashing my rx570 with the new fuzzy fan value set to 0, windows crashes on boot.  I had to restart in safe mode, disable the GPU, restart, reflash the GPU with the fuzzy fan set back to 1, and every thing worked again.  So even though 1.6.2 does save it, it will crash windows 10.  It would be awesome to be able to set sensible fan values in the bios, my rig runs on ubuntu, and setting the fans with sensors and bash scripts is a huge pain.  Without settings the fans in ubuntu though, the cards are all near 80c with the fans barely turning.  BTW great bios editor jaschaknack, thanks for making this.
